tachyon/crypto/sumcheck/multilinear/BUILD.bazel

```diff
@@ -0,0 +1,30 @@
+load("//bazel:tachyon_cc.bzl", "tachyon_cc_library", "tachyon_cc_unittest")
+
+package(default_visibility = ["//visibility:public"])
+
+tachyon_cc_library(
+    name = "multilinear_sumcheck",
+    hdrs = ["multilinear_sumcheck.h"],
+    deps = [
+        "//tachyon/base:range",
+        "//tachyon/crypto/sumcheck:sumcheck_prover",
+        "//tachyon/crypto/sumcheck:sumcheck_prover_msg",
+        "//tachyon/crypto/sumcheck:sumcheck_proving_key",
+        "//tachyon/crypto/sumcheck:sumcheck_verifier",
+        "//tachyon/crypto/sumcheck:sumcheck_verifier_msg",
+        "//tachyon/crypto/sumcheck:sumcheck_verifying_key",
+        "//tachyon/math/polynomials/multivariate:linear_combination",
```

Comment:
``` suggestion
        "//tachyon/crypto/sumcheck:sumcheck_prover",
        "//tachyon/crypto/sumcheck:sumcheck_verifier",
```
The deleted lines are already dependent on sumcheck_prover and sumcheck_verifier.